lmms_eval/models/model_utils/thyme/utils.py

```
@@ -0,0 +1,67 @@
+        "or reasoning. Give the answer to the following question directly.\n"
+        f'**User\'s Question:** "{user_question}"'
+    )
+    return prompt
+
+
+def generate_prompt_final_qa(user_question, user_image_path):
+    # Construct the prompt based on the given requirements
+    try:
+        with Image.open(user_image_path) as img:
+            user_image_size = f"{img.width}x{img.height}"
+    except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)